src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationExpressionTests.cs

```diff
@@ -230,6 +230,69 @@ public void NullableTypeTest_04()
             EOF();
         }
 
+        [Fact]
+        public void NullableTypeTest_05()
+        {
+            UsingStatement("ref object x = o1 is string ? ref o2 : ref o3;");
```

Comment:
Should `ref o1 is string ? ref o2 : ref o3` work too?

Reply:
Yes, that is normally how it would be used.  I'll be sure to test that.